content/influxdb/v2.0/reference/cli/influxd/upgrade/v2-dump-meta.md

```diff
@@ -0,0 +1,24 @@
+---
+title: influxd upgrade v2-dump-meta
+description: >
+  Output 2.0 boltdb metadata
+influxdb/v2.0/tags: []
+menu:
+  influxdb_2_0_ref:
+    parent: influxd upgrade
+weight: 301
+products: [oss]
+---
+
+The `influxd upgrade v2-dump-meta` command prints the contents of `influxdb.bolt` to the standard output.
```

Comment:
what does this help you do?

Reply:
Turns out these are developer tools. Removed.